polymer.externs.js

```diff
@@ -595,6 +595,15 @@ PolymerDomApi.prototype.setAttribute = function(attribute, value) {};
 /** @param {string} attribute */
 PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 
+/**
+ * @param {Function} callback
+ * @return {Object}
```

Comment:
Can we be more specific here? Based on https://github.com/Polymer/polymer/blob/master/src/lib/dom-api-effective-nodes-observer.html#L39, it looks like this could be {!{fn: !Function=, _nodes: !Array<!Node>}}. Same below.